### ISSUE TITLE
isr/config.yml" caused: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type.

### ISSUE THREAD

**Issue by hqmf8104** [idealo/image-super-resolution#166]
Hi Guys,

Running the following command:

docker run -v $(pwd)/data/:/home/isr/data -v $(pwd)/logs/:/home/isr/logs -v $(pwd)/weights/:/home/isr/weights -v $(pwd)/isr/config.yml:/home/isr/config.yml -it isr -t -d -c config.yml

causes this issue:

docker: Error response from daemon: OCI runtime create failed: container_linux.go:370: starting container process caused: process_linux.go:459: container init caused: rootfs_linux.go:59: mounting "/mnt/c/Users/patm2/Projects/sat_im_obj_detect/phase2.0/image-super-resolution/isr/config.yml" to rootfs at "/var/lib/docker/overlay2/3f25b8148d9372195b3b4ac587b948b360e14eeabf3f3bf159012eae805e4be7/merged/home/isr/config.yml" caused: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type.
ERRO[0000] error waiting for container: context canceled 

Any help would be greatly appreciated! Thanks!

**Comment by juulieen** [comment]
```shell
$ rm -rf isr/config.yml && cp ./config.yml ./isr/
```
did the trick for me

**Comment by jonathanmv** [comment]
changing the argument to mount the `config.yml` worked for me.
From `-v $(pwd)/isr/config.yml:/home/isr/config.yml`
To `-v $(pwd)/config.yml:/home/isr/config.yml`

So I point to the `config.yml` directly as there's not such file as `isr/config.yml`. Now I'm having [this issue](https://github.com/idealo/image-super-resolution/issues/163) instead.

**Comment by juulieen** [comment]
I did have this issue because I haven't downloaded the weights. Did you do it?

**Comment by jonathanmv** [comment]
Yes. Thanks for asking. I described my findings [here](https://github.com/idealo/image-super-resolution/issues/163#issuecomment-812038252)

**Comment by jonathanmv** [comment]
I created a [PR](https://github.com/idealo/image-super-resolution/pull/191) to address those issues
